package.json

```diff
@@ -107,7 +107,7 @@
     "url": "https://github.com/nasa/openmct.git"
   },
   "engines": {
-    "node": ">=14.19.1"
+    "node": ">=16.19.1"
```

Comment:
latest as of today

Reply:
fwiw there is `16.20.0` released on 3/28/23, if we want to take that instead?